### DISCUSSION TITLE
[Enhancement] Allow separator elements to operate as accordion fold

### DISCUSSION THREAD

**Discussion by trankillity** [Clooos/Bubble-Card#438] [Feature requests]
I would love to see the seperator elements have the option to be configured as an accordion element similar to [`fold-entity-row`](https://github.com/thomasloven/lovelace-fold-entity-row) or [Expander Card](https://github.com/Alia5/lovelace-expander-card).

This would allow for even tighter control of the elements in a pop-up as you could have them collapsed down to make the pop-up cleaner.

You would need to a way to make the subsequent cards children of the separator element, which I guess would turn it more into a "card group" element than a strict separator - I just like the styling of the separator and feel it would work well as an accordion fold.

**Comment by aredon** [comment]
I would second this. There are many things (charts) that I would much rather tuck into a collapsing card than place in a pop-up. 

It can _kind of_ be done with: [This Expander Card Fork](https://github.com/MelleD/lovelace-expander-card) while using the separator card as a title card but it's a bit clunky. Would be better to have it as a native feature - especially since buttons are already included.
